source/Halibut.Tests/Transport/Streams/NetworkTimeoutStreamFixture.cs

```diff
@@ -187,7 +187,7 @@ public async Task WriteAsyncShouldCancel()
                     stopWatch.Stop();
 
                     actualException.Should().NotBeNull().And.BeOfType<OperationCanceledException>();
-                    actualException!.Message.Should().Be("The WriteAsync operation was cancelled.");
+                    actualException!.Message.Should().Be($"The {nameof(NetworkTimeoutStream)}.WriteAsync operation was cancelled.");
```

Comment:
I don't know if we want this change. Out stream is the thing forcing the cancel but it's a RadAsync operation on a different stream that is doing the work. I don't know if it makes sense to name the class which happened to have the timeout logic wrapper.

Reply:
The motivation was that I didn't want to lose context - we previously would be able to trace this error back to `NetworkTimeoutStream`, but generalising means the call stack wouldn't be as clear as there are now multiple usages, so I thought the caller class name in the error might help.

I'm happy to remove it though, it was never strictly necessary anyway.